test/unit/services/services_spec.js

```diff
@@ -0,0 +1,121 @@
+"use strict";
+
+var Instance = require("../../../lib/services/services-db");
+var _        = require("lodash");
+var expect   = require("chai").expect;
+var Genesis  = require("genesis");
+var Bluebird = require("bluebird");
+var Sinon    = require("sinon");
+require("sinon-as-promised")(Bluebird);
+
+describe("the Adapter class ", function () {
+	it("is frozen", function () {
+		expect(Object.isFrozen(Instance), "frozen").to.be.true;
+	});
+
+	function describeCreate (description, options, fails) {
+		var optionsToPass = JSON.parse(JSON.stringify(options));
```

Comment:
This is a rather confusing way to make an object clone. Check out `_.clone`! Lodash has a lot of nice utility functions like that.